backend-go/internal/core/auth/server.go

```
@@ -2,11 +2,11 @@ package auth
 
 import (
 	"github.com/labstack/echo/v4"
+	"github.com/yorkie-team/codepair/backend/internal/jwt"
```

golangci-lint (1.64.8)

File is not properly formatted

(goimports)